lib/commands/screenshots.js

```diff
@@ -54,41 +51,34 @@ commands.getScreenshot = async function () {
   };
 
   try {
-    if (this.isRealDevice() && await isIdevicescreenshotAvailable()) {
-      log.debug(`Taking screenshot with 'idevicescreenshot'`);
-      return await getScreenshotWithIdevicelib(this.opts.udid);
+    if (this.isRealDevice()) {
+      if (await isIdevicescreenshotAvailable()) {
```

Comment:
If the user does not have idevicescreenshot installed and is on a real device, this will now do nothing.

Reply:
why? it will fall back to _retryInterval_ below

Reply:
also will print the message about the tool is not present

Reply:
Oh! I missed the move of the retry outside of the catch block. Sorry.